config/simd/m128.hpp

```diff
@@ -0,0 +1,240 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+
+#ifndef MFEM_TEMPLATE_CONFIG_SIMD_M128
+#define MFEM_TEMPLATE_CONFIG_SIMD_M128
+
+#include "../tconfig.hpp"
+
+template <typename scalar_t> struct AutoSIMD<scalar_t,2,2>
```

Comment:
Does this make sense with types other than `scalar_t = double`? Maybe this should be a full specialization:
```c++
template <> struct AutoSIMD<double,2,2>
```

Reply:
We could simplify, yes.

Reply:
`done`